code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -477,6 +490,36 @@
 				visible_message("\icon[src] <span class='notice'>The [name] powers down and goes silent.</span>")
 				update_icon()
 
+		if("toggle_alert")
+			if(!alerts_on)
+				user.visible_message("<span class='notice'>[user] activates [src]'s alert notifications.</span>",
+				"<span class='notice'>You activate [src]'s alert notifications.</span>")
+				visible_message("\icon[src] <span class='notice'>The [name] buzzes in a monotone voice: 'Alert notification system initiated'.</span>'")
+				alerts_on = TRUE
+				update_icon()
+			else
+				alerts_on = FALSE
+				user.visible_message("<span class='notice'>[user] deactivates [src]'s alert notifications.</span>",
+				"<span class='notice'>You deactivate [src]'s alert notifications.</span>")
+				visible_message("\icon[src] <span class='notice'>The [name] buzzes in a monotone voice: 'Alert notification system deactivated'.</span>'")
+				update_icon()
+
+		if("toggle_radial")
+			if(!alerts_on)
```

Comment:
I assume here you wanted `if(!radial_mode)`, but you can also apply the same mentioned logic.